jrnl/path.py

```diff
@@ -0,0 +1,13 @@
+import os.path
+
+
+def home_dir():
+    return os.path.expanduser("~")
+
+
+def expand_path(path):
```

Comment:
I think we could do with just `home_dir` and `expand_path` as long as `expand_path` also runs `abspath`. Would you mind merging `expand_path` and `absolute_path` into one function?

Reply:
I tried that at first because I assumed that it wouldn't make a difference but I think that's not a great idea for a couple reasons.

1. It breaks the tests as currently written. A few BDD tests check the console output of the application and the output that gets parsed contains relative paths. That means that those tests would have to changed to parse the absolute path which is a bit more complicated.
2. The absolute path is only needed once so it would add extra work to all the other calls.
3. Console error messages are more readable when they display the relative path instead of the absolute path.